<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="Display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
backgrounnd-color:gray;
border:1px solid gray;

}</string>
       </property>
       <property name="text">
        <string notr="true">0.0</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="2">
        <widget class="QPushButton" name="ButtonB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="Button1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Button2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="Button3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Button7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="Button6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Button5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Button4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Button8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="Button9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="Button0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="ButtonC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="ButtonE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="ButtonT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="Buttone">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="NN">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(192, 192, 192);
	border:1px solid gray;
	pandding:5px;
}
QpushButton:pressed{
	background-color:#A9A9A9;
	border:1px solid gray;
pandding:5px
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Temizle">
       <property name="text">
        <string>temizle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
